samples/boards/arduino_101/environmental_sensing/README.rst

```diff
@@ -25,7 +25,7 @@ To use this sample, the following hardware is required:
 * Arduino 101
 * `Grove LCD module`_
 * `Grove Base Shield`_ [Optional]
-* `HDC1008 Sensor`_
+* `TI_HDC Sensor`_
```

Comment:
It is still HDC1008 sensor. And what has it to do with _samples: added ti_hdc_ ?

Reply:
Noted